fedora-spec-files/generate-data.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/python3
+""" Generate %patch, %source and %prep data about most patched Fedora spec files. """
+import json
+import pathlib
+import re
+from typing import Dict, Tuple
+from bodhi.client.bindings import BodhiClient
+
+# rpm-specs-latest.tar.xz unpacked in /
+RPM_SPECS = pathlib.Path("/rpm-specs")
+# process this many packages ordered by the amount of patches
+PROCESS_PACKAGE_COUNT = 50
```

Comment:
Could this be moved into the notebook, so that data for all packages is generated, but it's possible to control from the notebook how much of this data to visualize?

Reply:
that won't work because we compute the number of builds per release from koji, which takes a few seconds for every package - doing such action for all packages would take hours

also, anyone can change that value easily, it's a simple script - let's not overcomplicate

Reply:
The :-1: meant that it's unfortunate that it's not possible to change this and make analyzing the data more flexible.

Reply:
roger, thanks for explanation